plugins/badges/src/components/EntityBadgesCard.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { useAsync } from 'react-use';
+import { Typography } from '@material-ui/core';
+import { CodeSnippet, InfoCard, useApi } from '@backstage/core';
+import { ENTITY_DEFAULT_NAMESPACE } from '@backstage/catalog-model';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { badgesClientApiRef } from '../BadgesClientApi';
+
+export const EntityBadgesCard = () => {
+  const badgesClientApi = useApi(badgesClientApiRef);
```

Comment:
yep as mentioned below, should be more like

```suggestion
  const badgesApi = useApi(badgesApiRef);
```

Reply:
I think I went with it that way, to not confuse with the BadgesApi from the backend plugin. But if it's ok to re-use the plugin name in the frontend plugin.. ?

Reply:
Oh. Yeah should be OK right, as they don't import each other. I think I suggested naming the backend one something with "builder" too right? 🤔